include/wpe/gamepad.h

```diff
@@ -358,6 +358,20 @@ void wpe_gamepad_set_client(struct wpe_gamepad*, const struct wpe_gamepad_client
 WPE_EXPORT
 const char* wpe_gamepad_get_id(struct wpe_gamepad*);
 
+/**
+ * wpe_gamepad_dispatch_analog_button_changed:
+ * @gamepad: opaque gamepad object.
+ * @button: the analog button that changed its value.
+ * @value: the new analog @button value.
+ *
+ * Method called by application (gamepad implementator). It reports to
+ * WPEWebkit a change in the value  of analog @button.
+ *
+ * Since: 1.15
```

Comment:
Version 1.15 will be a development one, the first stable version to include this will be 1.16.0 so it's better to write `1.16` here.